src/main/java/io/sentry/demos/example/Application.java

```diff
@@ -45,11 +46,11 @@ private void checkout(List<Item> cart) {
 		inventory = tempInventory;
 	}
 
-	// Class constructor to initialize logging 
 	public void Application() {
 
 	}
 
+	@CrossOrigin(origins = "http://localhost:5000")
```

Comment:
The front end call kept getting denied about a 403 CORS problem until I added this.